cmd/discovery5.go

```diff
@@ -41,14 +47,46 @@ type Eth2InfoData struct {
 	OutputPath    string
 }
 
+func GeneratePrivKey() string {
```

Comment:
I'm not sure why do we need two functions that Generate a `ecdsa.PrivateKey`?

As far as I know, the Ethereum Nodes only need the `*ecdsa.PrivateKey` to sign messages. Therefore:
```go
func GenNewPrivKey() (*ecdsa.PrivateKey, error) {
	return ecdsa.GenerateKey(gcrypto.S256(), rand.Reader)
```
should work